apps/search/scripts/migrations/README.md

```diff
@@ -0,0 +1,7 @@
+# Webhook migration scripts
+
+Test migration with dry run, operation will not modify any data:
+`npx tsx scripts/migrations/run-webhooks-migration-dry-run.ts`
+
+To start the migration run command:
+`npx tsx scripts/migrations/run-webhooks-migration.ts`
```

Comment:
note for upcoming hackathon solution - probably some npm scripts will do the job